app/api/dao/mentorship_relation.py

```diff
@@ -158,30 +151,25 @@ def accept_request(user_id, request_id):
         return {'message': 'Mentorship relation was accepted successfully.'}, 200
 
     @staticmethod
+    @email_verification_required
     def reject_request(user_id, request_id):
-
         user = UserModel.find_by_id(user_id)
-
-        # verify if user exists
-        if user is None:
-            return {'message': 'User does not exist.'}, 404
-
         request = MentorshipRelationModel.find_by_id(request_id)
 
         # verify if request exists
-        if request is None:
+        if request == None:
             return {'message': 'This mentorship relation request does not exist.'}, 404
 
         # verify if request is in pending state
-        if request.state is not MentorshipRelationState.PENDING:
+        if request.state != MentorshipRelationState.PENDING:
             return {'message': 'This mentorship relation is not in the pending state.'}, 400
 
-        # verify if I'm the receiver of the request
-        if request.action_user_id is user_id:
+        # verify if both sender and reciver are same
```

Comment:
remove

Reply:
@ramitsawhney27 what you want me to remove? ```# verify if both sender and receiver are same```comment? or do you want me to change it to something else? Please specify